tests/src/test/java/com/oracle/weblogic/imagetool/tests/ITImagetool.java

```diff
@@ -881,4 +882,59 @@ void createWlsImgWithAdditionalBuildCommands(TestInfo testInfo) throws Exception
         assertTrue(inspect.stdout().contains("final-build-commands:finalBuildCommands"),
             tagName + " does not contain the expected label");
     }
+
+    /**
+     * Create a WLS image using Java Server JRE image as a base.
+     * This tests that the JAVA_HOME is correctly identified and applied in the CREATE.
+     *
+     * @throws Exception - if any error occurs
+     */
+    @Test
+    @Order(26)
+    @Tag("nightly")
+    @DisplayName("Create image with WLS using Java ServerJRE")
+    void createImageWithServerJRE(TestInfo testInfo) throws Exception {
+        String tagName = build_tag + ":" + getMethodName(testInfo);
+        String command = new CreateCommand()
+            .tag(tagName)
+            .fromImage("container-registry.oracle.com/java/serverjre:8u271")
```

Comment:
Sorry do not kill me :-( can you use the tag 8 so that we get the latest and do not need to make a change every quarter after there is a new version.